### DISCUSSION TITLE
problem with pumgen to convert the mesh file?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#706] [Q&A]
I want to try a new model like the belong picture, and I define  a physical volume, and I genetate the mesh file, I compared it with the WH2_LOH3, and I find they have the same format, but when I use `pumgen head.msh -s msh2` to convert the format, it has no response,do you know why this happen? thanks for your help in advance. 

![image](https://user-images.githubusercontent.com/114474894/197148673-212900e9-351a-48b0-8755-a9454486cbf6.png)

![image](https://user-images.githubusercontent.com/114474894/197149018-cfe6cb94-81bd-4c5e-ad34-118a32012e1d.png)

the same format
![image](https://user-images.githubusercontent.com/114474894/197150688-97865e7b-1c64-4d7b-ac88-27eb490ad136.png)

![image](https://user-images.githubusercontent.com/114474894/197150824-5c41dbb0-22ec-4b77-a674-4393e38412b5.png)

**Comment by sebwolf-de** [comment]
Did you define `Physical Surface`s for all surface elements?

**Comment by sebwolf-de** [comment]
Also, please rather post the whole mesh or geo file and not a screenshot.

**Reply by buzhiqiuyuebai** [reply]
> Also, please rather post the whole mesh or geo file and not a screenshot.
 
I use gmsh open the file `Construction - Head5.51.igs`, and use `optimize 3D`  to genetate the file `Construction - Head5.51 - before.msh`, then I open the file  `Construction - Head5.51 - before.msh` , and geometry-->physical groups-->add volume and surface to add physical volume and physical surface, and finally save mesh to get the file `Construction - Head5.51.msh`, when I use pumgen to convert the file  `Construction - Head5.51.msh`, it doesn't work. 
The file I mentioned before can find in the link:
https://github.com/buzhiqiuyuebai/head-file
